README.md

```diff
@@ -362,6 +362,16 @@ Please make sure [the common prerequisites](#prerequisites) as well as [the prer
 Then PyTorch can be built with the command:
 
 ```cmd
+:: Set the CMAKE_PREFIX_PATH to help find corresponding packages
+:: %CONDA_PREFIX% only works after `conda activate custom_env`
+:: Use `$env:CMAKE_PREFIX_PATH = "${env:CONDA_PREFIX};${env:CMAKE_PREFIX_PATH}"` for powershell
+
+if defined CMAKE_PREFIX_PATH (
+    set "CMAKE_PREFIX_PATH=%CONDA_PREFIX%\Library;%CMAKE_PREFIX_PATH%"
+) else (
+    set CMAKE_PREFIX_PATH=%CONDA_PREFIX%\Library
```

Comment:
Please add quotation marks here in case there is whitespace issue in `%CONDA_PREFIX%`.
```suggestion
    set "CMAKE_PREFIX_PATH=%CONDA_PREFIX%\Library"
```